lib/list-swipe-save.js

```diff
@@ -59,7 +59,7 @@ SwipeSaveList.prototype.init = function(){
     actionThreshold : 120
 	});
 
-  Fliplet.Storage.get('saved_list_' + _this.listSwipe.container.parentNode.dataset.listId, {
```

Comment:
`Fliplet.Widget.getUUID(widgetInstanceId)` is definitely a better method. The current method relies too much on a set DOM structure. I'll make note of it next time I update the widget ;)